index.Rmd

```diff
@@ -31,6 +31,7 @@ Such table data was merged in R with county polygon data provided through the TI
 ## Data: 
 The data I extracted is made available at: [https://github.com/GEO511-2021/2021_project-MicaelaFLipman/blob/ea0119a7c180fabed5d48131db971a109c57de23/LongForm.csv]
 
+
```

Comment:
I know you have mentioned how you are going to use your data, but It would be great if you can briefly describe your datasets, e.g. where did you get them from :).